examplemusig2.c

```diff
@@ -175,8 +187,9 @@ int main(void) {
 
     /**** Verification ****/
     /* Verify the aggregated signature with secp256k1_schnorrsig_verify */
-    if (musig2_ver_musig(ctx, signature2, mca2.aggr_pk, MSG_2, MSG_2_LEN))
+    if (secp256k1_schnorrsig_verify(ctx, signature2, MSG_2, MSG_2_LEN, &xonly_aggr_pk)) {
```

Comment:
What if different verifiers will verify the MuSig2 for different states? It does not affect any functionality, but I think it is better to indicate that the verifier should aggregate public keys before verification, for the sake of integration of the library, WDYT?